components/_patterns/02-molecules/date-bars/date-bar.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Event address bar for displaying date and time information for events.
+ *
+ * Available variables:
+ * - ended: (optional) Boolean. If true, display string saying event has ended.
+ */
+#}
+
+{% set card_base_class = 'date-bar' %}
+
+<ul {{ bem(card_base_class, (card_modifiers), card_blockname) }}>
```

Comment:
Hello! I think the requirements for this might have gotten mixed up with the requirements we're discussing for the location hours. We were discussing those as a list since they are currently a table. In any case, these can just be text (no list).

We did also have a conversation around if these should be headings or not based on their usage in OSEA - Section_Event Address and Actions. But believe we were going to resolve with changes to that organism rather than to the molecule.

Reply:
@wlla Sounds like my mistake, then. I'll update this code so that it's no longer a list.

Willa, it seems like these are only used in OSEA-Section_Event Address and Actions, which will affect the way that @ModulesUnraveled builds the organism component. What header level should these be in at in that context?